tests/serialized_test_data_generation/Makefile

```diff
@@ -84,18 +84,19 @@ run_model: ## run model using Docker image to generate serialized data
 	cp $(EXPERIMENT_DIR)/$(EXPERIMENT).yml $(DATA_DIR_HOST)
 	cp $(EXPERIMENT_DIR)/$(EXPERIMENT).yml $(DATA_DIR_HOST)/input.yml
 
-pack_data: ## pack *.dat files into a *.tar.gz and delete them
+pack_data: ## convert to netcdf files, pack *.dat files into a *.tar.gz and delete them
 	@echo ""
 	@echo ">> Packing data in $(DATA_DIR_HOST) to $(TARFILE_HOST)"
 	@if [ -f $(TARFILE_HOST) ] ; then \
 		echo "Error: Tar file $(TARFILE_HOST) already exists. Already packed?" ; \
 		exit 1 ; \
 	fi
+	docker run --mount type=bind,source=$(shell pwd)/serialbox_to_netcdf.py,target=/root/serialbox_to_netcdf.py -v $(DATA_DIR_HOST):/data -it $(COMPILED_IMAGE) bash -c "pip3 install xarray==0.16.2 f90nml==1.1.0 scipy==1.5.4 && python3 /root/serialbox_to_netcdf.py /data /data"
```

Comment:
Do we also want to compress netcdf files? Some of the larger dataset is already pushing the limit of our docker storage size, so it may or may not be feasible to compress.

Reply:
I think it's very good to have the flexibility to download/inspect a single savepoint file from GCS. As I understand it this is a very, very small fraction of our overall google cloud storage (which is in the terabytes).